grpc.opam

```diff
@@ -17,6 +17,7 @@ depends: [
   "ppx_deriving"
   "ppx_yojson_conv"
   "ppx_let"
+  "fmt"
```

Comment:
```suggestion
  "fmt" { with-test }
```

Reply:
Sorry I myself misunderstood that `fmt` is not for test but for example😜

```suggestion
```

In addition, Dune automatically detects the build dependencies, so we dont't need `dune build` before `dune runtest`.